### ISSUE TITLE
write installation docs

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg-import#15]
see the borgbackup docs, use same style, maybe even copy from there and modify as needed.

**Comment by gobborg** [comment]
Can this issue be closed due to [PR#52](https://github.com/borgbackup/borg-import/pull/52)?

**Comment by ThomasWaldmann** [comment]
Yes, there are some basic install docs, so we can close this for now.
